scripts.js

```diff
@@ -50,3 +50,20 @@ if (sidebar) syncSidebar(sidebar)
 if (docSearch) initSearch()
 if (themeToggle) handleThemeSwitcher(Boolean(isDark))
 if (isDark) updateTheme(isDark)
+
+const isCute = () => {
+  if(!location.search) return null
+  const params = location.search.match(/^\?(.*)$/)[1].split('&')
```

Comment:
Please consider using the [URLSearchParams](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams) constructor instead of your custom logic